state-chain/pallets/cf-ingress-egress/src/lib.rs

```diff
@@ -579,7 +579,7 @@ pub mod pallet {
 			asset: TargetChainAsset<T, I>,
 		},
 		BatchBroadcastRequested {
-			broadcast_id: BroadcastId,
+			broadcast_ids: Vec<BroadcastId>,
```

Comment:
This will need a bit more thought, I don't think it's as simple as it seems. 

The event is used to correlate each individual broadcast with a swap egress. A many-to-many mapping doesn't really work for this. So this event needs to remain unchanged. 

We might have to pass in the egress id for each transfer with the transfer params, and the `new_unsigned` would return a `Vec<(Vec<EgressId>, Transaction)>`...

Reply:
Good point, I also wondered about this.

Reply:
Updated the AllBatch signature accordingly. 
The broadcast event now only include EgressIds that a associated with the broadcast